site/docs/v1/tech/apis/_import-users-request-body.adoc

```diff
@@ -103,7 +103,11 @@ The User's mobile phone number. This is useful is you will be sending push notif
 [field]#users``[x]``.password# [type]#[String]# [required]#Required#::
 The User's password. If [field]#encryptionScheme# is provided this value is the existing password hash, if [field]#encryptionScheme# is not provided this value is assumed to be plain text and it will be hashed during import.
 +
-[field]#password# should contain only the password hash as a base64 encoded string. Depending on how your password database stores hashes, additional processing may be required. Please see the <<Password Hashes>> section for more information about importing particular password hashes.
+[field]#password# should contain only the password hash.
+Unless you are using a custom plugin that does not utilize base64, [field]#password# should be base64 encoded.
+It is recommended to use a base64 encoded string, but if you are importing hashes from a legacy system that uses base16, base32, or another encoding and are writing a custom plugin, import the hash in the same format the plugin produces.
++
+Please see the <<Password Hashes>> section for more information about importing particular password hashes.
```

Comment:
In most cases we don't add line returns for readability, but only to add spacing in docs. I think this will just render all on the same line? If so, we should remove the line returns.

Reply:
Per discussion in slack, going to revert this and keep a single paragraph on one line.